referenceframe/worldstate.go

```diff
@@ -102,6 +103,30 @@ func (ws *WorldState) ToProtobuf() (*commonpb.WorldState, error) {
 	}, nil
 }
 
+// String returns a string representation of the geometries in the WorldState.
+func (ws *WorldState) String() string {
+	if ws == nil {
+		return ""
+	}
+
+	t := table.NewWriter()
+	t.AppendHeader(table.Row{"Name", "Geometry Type", "Parent"})
+	for _, geometries := range ws.obstacles {
+		for _, geometry := range geometries.geometries {
+			name := geometry.Label()
+			stringRep := geometry.String()
+			parent := geometries.frame
```

Comment:
[opt] could avoid the extra assignment step by assigning them in the AppendRow function call